docker/README.md

```diff
@@ -10,5 +10,13 @@ make docker
 
 in the project root.
 
+To start entire app, run
+
+```
+make docker-compose
```

Comment:
Are we default to redpanda? In the later development, we may need to test multiple source systems. So maybe we should change the name to:

```
make docker-compose-redpanda
```
`docker-compose-kafka`, `docker-compose-pulsar`, etc.

Reply:
I think we can do it when add other source.